appcompat/appcompat/src/androidTest/java/androidx/appcompat/widget/AppCompatTextViewTest.java

```diff
@@ -1299,6 +1299,10 @@ public void onDestroyActionMode(ActionMode mode) {
     @RequiresApi(api = Build.VERSION_CODES.P)
```

Comment:
You should use `@SdkSupress(minSdkVersion=` here

Reply:
thanks , i got it.